app/views/shared/_footer.html.haml

```diff
@@ -131,21 +130,22 @@
           %img{src: ContentConfig.footer_logo.url, width: "220"}
       .small-12.medium-5.columns.text-left
         %p.text-small
-          = t :footer_legal_call
+          = t '.footer_legal_call'
           %a{href: ContentConfig.footer_tos_url}
-            = t :footer_legal_tos
+            = t '.footer_legal_tos'
           &#124;
-          = t :footer_legal_visit
+          = t '.footer_legal_visit'
           %a{href:"https://github.com/openfoodfoundation/openfoodnetwork", target: "_blank"} GitHub
         %p.text-small
-          = t :footer_legal_text_html, {content_license: link_to('CC BY-SA 3.0', 'https://creativecommons.org/licenses/by-sa/3.0/'), code_license: link_to('AGPL 3', 'https://tldrlegal.com/license/gnu-affero-general-public-license-v3-(agpl-3.0)')}
+          = t '.footer_legal_text_html', {content_license: link_to('CC BY-SA 3.0', 'https://creativecommons.org/licenses/by-sa/3.0/'), code_license: link_to('AGPL 3', 'https://tldrlegal.com/license/gnu-affero-general-public-license-v3-(agpl-3.0)' )}
         %p.text-small
           %div
-            = t :footer_legal_data_text
-            = t :footer_legal_data_call
-            %a{'cookies-policy-modal'=> true, 'cookies-banner'=> !CookiesConsent.new(cookies, request.host).exists? }
-              = t :footer_legal_data_cookies_policy
-
+            - cookies_policy_link = link_to( t( '.footer_data_cookies_policy' ), '', 'cookies-policy-modal' => true, 'cookies-banner' => !CookiesConsent.new(cookies, request.host).exists? && Spree::Config.cookies_consent_banner_toggle)
+            - privacy_policy_link = link_to( t( '.footer_data_privacy_policy' ), Spree::Config.privacy_policy_url, :target => '_blank' )
+            - if Spree::Config.privacy_policy_url.present?
+              = t '.footer_data_text_with_privacy_policy_html', {cookies_policy: raw( "#{cookies_policy_link}" ), privacy_policy: raw( "#{privacy_policy_link}" )}
```

Comment:
`html_safe` seems to be the better option instead of `raw` here:
```ruby
cookies_policy_link.html_safe
```

Reply:
interesting, apparently raw = to_s.html_safe.
Ive changed it to html_safe. Much better! Thanks.